src/Avalonia.Base/Styling/Container.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using Avalonia.Controls;
+using Avalonia.PropertyStore;
+
+namespace Avalonia.Styling
+{
+    /// <summary>
+    /// Defines a container.
+    /// </summary>
+    public class Container : StyleBase
```

Comment:
The name feels to be too generic and is likely to cause name clashes later. We already have IContainer that means a completely different thing.